cmd/cvc-operator/controller/cstorvolumeclaim.go

```diff
@@ -521,6 +529,92 @@ func randomizePoolList(list *apis.CStorPoolInstanceList) *apis.CStorPoolInstance
 	for _, randomIdx := range perm {
 		res.Items = append(res.Items, list.Items[randomIdx])
 	}
-
 	return res
 }
+
+// getOrCreatePodDisruptionBudget will does following things
+// 1. It tries to get the PDB that was created among volume replica pools.
+// 2. If PDB exist it returns the PDB.
+// 3. If PDB doesn't exist it creates new PDB(With CSPC hash)
+func getOrCreatePodDisruptionBudget(
+	cvObj *apis.CStorVolume, cspcName string) (*policy.PodDisruptionBudget, error) {
+	poolNames, err := cvr.GetReplicaPoolNames(cvObj)
```

Comment:
cvr.GetReplicaPoolNames(cvobj) doesn't makes sense and can lead to circular dependency.. make it as cvobj.GetReplicaPoolNames()

Reply:
It will not lead to circular dependency but passing cvObj in CVR package doesn't make sense I will update it.

Reply:
updated it as `cvr.GetVolumeReplicaPoolNames(pvName, namespace)`